webshopping/lolshopping/templates/design/home.html

```diff
@@ -49,12 +49,12 @@
     <div class="row" id="results">
 
         {% for champs in all %}
-        <div class="col-lg-3 abc" id = "contain">
+        <div class="col-lg-3 abc" id="contain">
             <img class ="thumbnail" src="{{champs.imageURL}}">
             <div class="box-element product">
-            <span class="namez">
+            <span class="class_name">
```

Comment:
Why is this called "class_name"? Naming this class as "name" is obviously not very descriptive, but looking at line 56, this can clearly be called "champ_name".

Reply:
yeah "champ_name" is better, I will change right now. thanks